src/components/cardContent/index.js

```diff
@@ -1,11 +1,8 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-function ProfileContent({ postContent }) {
-  const {
-    title, detail, image,
-    description, viewCount
-  } = postContent;
+const ProfileContent = ({ postContent }) => {
+  const { title, detail, image, description, viewCount } = postContent;
```

Comment:
postContent can be undefined at some point of time which will break the code so it is important to add **|| {}** when destructing.
const { title, detail, image, description, viewCount } = postContent || {};